docs-ref-mapping/reference-latest.yml

```diff
@@ -420,7 +420,6 @@
       landingPageType: Service
       children:
       - com.azure.messaging.eventgrid*
-      - com.microsoft.azure.eventgrid*
```

Comment:
why are we removing this?

Reply:
This package name is for T1. But this is reference-latest.yml.

Reply:
So, shouldnt this go into the reference-legacy file then?
Regardless, can we keep this for a few more months until we are done with the reliability improvements and mark T1 as deprecated?

Reply:
Added back.